### ISSUE TITLE
redundant folders after git pull

### ISSUE THREAD

**Issue by fabeit** [EDmodel/ED2#165]
Just did a new installation and after a git pull of the master branch just following the instruction on the wiki I have a weird folder structure. In the first level I have all the folder and files, but then I have an additional ED2 folder that inside has all the same folder but doesn't have the Doxyfiles and doesn't have the EDR folder. If I another level down into the second ED folder there are the usual folders found inside ED in addition to a dbgbuild folder.

I am confused, is there anything wrong with my github setup or has there been some changes? Why these redundant folders and which of the two ED folders to work with?

thanks

**Comment by mdietze** [comment]
In terms of the dgbbuild, that was a change @DanielNScott made recently so that there would be separate builds for debugging vs optimized compilation.

As for an additional ED2 folder, that's not part of the git master -- I wonder if you accidentally did a second checkout within the first?

**Comment by mdietze** [comment]
@DanielNScott you probably want to update the wiki documentation on installation to explain the new build

**Comment by fabeit** [comment]
@mdietze I am sure I have run the git pull command only once and in any case the ED folder in the root directory has a different folder than the ED folder under ED2. Maybe I am pulling my fork and then the master ?

**Comment by DanielNScott** [comment]
Good point about the documentation @mdietze thanks for mentioning it. Although I think you mispoke about the debug build situation. Just to be clear, there were previously dbgbuild and build folders, and dbgbuild had, more or less, slightly tweaked copies of everything in build. Now there is only one build folder, and it actually takes advantage of the options that are present within the include.mk files.
